modules/ROOT/pages/managing-users.adoc

```diff
@@ -3,31 +3,85 @@ ifndef::env-site,env-github[]
+Manage portal members using the *Portal members* page. When a guest user's registration is approved, the user's name is added to the list of members. You can view a searchable list of current members and manage their accounts by enabling or disabling them and adding them to user groups.
 
 === View and Search for Members 
-Find members by entering the member's name or email address in the *Search by name or email* box. The member is listed in the search results.
+Find members by entering the member's name or email address in the *Search by name or email* box. 
 
 === Enable or Disable Access to Your Portal
 When a user's access to the portal is approved in the *Pending registration requests* page, the *Enable portal access* slider is enabled in the *Portal members* page. Disable a user's access by moving the slider to off.
 
+=== Assign Members to User Groups
+Assign users to user groups to manage which APIs they can see in your portal. You can assign a user to multiple user groups or change user groups.
+
+. From the *Portal members* page, locate the member's name in the list and click *View and assign user groups*.
+. Select one or more user groups and click *Apply*.
+
 == Manage Pending Registration Requests
-Use the *Pending registration requests* page to approve or reject requests for users who registered to access the portal. You can view the user's name, email, registration date, and registration status of the users who register for access. The maximum pending user registration requests is 500.
+Use the *Pending registration requests* page to approve or reject requests for users who registered to access the portal. You can view the user's name, email, registration date, and registration status of the users who register for access. By default, users are assigned to the Members profile. You can add them to the available user groups that can see specific APIs in the portal. The maximum pending user registration requests is 500.
 
 When the registration is approved, the user is removed from the list on the *Pending registration requests* page and added to the list of members on the *Portal members* page. 
 
 If the registration is rejected, the user is removed from the list and can't log in and access the portal.
 
 === Approve or Reject Member Registration
-After a guest user registers for an account using self-registration, approve or reject the request. 
+After a guest user registers for an account using self-registration, approve or reject the requests and assign the user to user groups. All users who self-register are assigned as members. You can assign member users to multiple user groups.
 
 . Click *Pending registration requests*.
 . To find a user, enter the user's name or email in the *Search by name or email* box.
+. To assign the user to a user group:
+.. Click *Members* and select one or more user groups.
+.. Click *Apply*.
 . Click *Approve* or *Reject*.
 
+== Manage User Groups
```

Comment:
These H3 sections are all super short. I wonder if there's a way to streamline into a DL or something.